README.md

```diff
@@ -187,12 +187,30 @@ infixl 4 <*>, <*, *>, <**>
 It uses exactly the same syntax as usual Haskell fixity declarations to make
 it easier for Haskellers to edit and maintain.
 
-Besides, all of the above-mentioned parameters can be controlled from the
+As of Ormolu 0.7.0.0, `.ormolu` files can also contain instructions about
+module re-exports that Ormolu should be aware of. This might be desirable
+because at the moment Ormolu cannot know about all possible module
+re-exports in the ecosystem and only few of them are actually important when
+it comes to fixity deduction. In 99% of cases the user won't have to do
+anything, especially that most common re-exports are already programmed into
```

Comment:
Do we want to explicitly encourage people to create issues when they encounter missing re-exports they think are useful for many users?

Reply:
I added a sentence to encourage such contributions :slightly_smiling_face: